### ISSUE TITLE
[sdk-python] Can't set negative int at value_size_delta of accountMetadata

### ISSUE THREAD

**Issue by xembook** [symbol/symbol#7]
I want to update account metadata ,
but I cant set negative int at value_size_delta of accountMetadata.

for example, 
```python
aliceTx = facade.transaction_factory.create_embedded({
    'type': 'accountMetadata',
    'signer_public_key': alicePubkey,
    'target_address': bobAddress,
    'scoped_metadata_key': metadataKey,
    'value_size_delta': -1,
    'value': unhexlify(xorValue)
})
```
error message is 
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\user\AppData\Local\Programs\Python\Python38-32\lib\site-packages\symbol_catbuffer\EmbeddedAccountMetadataTransactionBuilder.py", line 139, in serialize
    bytes_ = GeneratorUtils.concat_typed_arrays(bytes_, self.body.serialize())  # kind:CUSTOM
  File "C:\Users\user\AppData\Local\Programs\Python\Python38-32\lib\site-packages\symbol_catbuffer\AccountMetadataTransactionBodyBuilder.py", line 103, in serialize
    bytes_ = GeneratorUtils.concat_typed_arrays(bytes_, GeneratorUtils.uint_to_buffer(self.value_size_delta, 2))  # serial_kind:SIMPLE
  File "C:\Users\user\AppData\Local\Programs\Python\Python38-32\lib\site-packages\symbol_catbuffer\GeneratorUtils.py", line 38, in uint_to_buffer
    return uint.to_bytes(buffer_size, byteorder='little', signed=False)
OverflowError: can't convert negative int to unsigned
```
by the way,
when I set 0xFFFF instead of  `-1`, it seems that value_size_delta has 65535

**Comment by Jaguar0625** [comment]
Needed to redo the fix in 915bce0f7.

**Comment by cryptoBeliever** [comment]
Link #37
